src/builder-module.c

```diff
@@ -1788,7 +1788,7 @@ builder_module_build_helper (BuilderModule  *self,
           if (libdir)
             g_ptr_array_add (configure_args_arr, g_strdup_printf ("-DCMAKE_INSTALL_LIBDIR:PATH='%s'", libdir));
```

Comment:
This `libdir` can also be an absolute path.

Reply:
That's user input though, if they set wrong values they will get weird behaviour back.

Reply:
The docs say the default is `/app/lib`. It was clearly intended to support absolute paths and it should be fine to accommodate both.

Reply:
So what to do? You can already set that to an absolute path if required for some reason in the manifest.

Reply:
```c
          if (libdir && g_path_is_absolute (libdir))
            g_ptr_array_add (configure_args_arr, g_strdup_printf ("-DCMAKE_INSTALL_FULL_LIBDIR:PATH='%s'", libdir));
          else if (libdir)
            g_ptr_array_add (configure_args_arr, g_strdup_printf ("-DCMAKE_INSTALL_LIBDIR:PATH='%s'", libdir));
```

Reply:
Note that setting CMAKE_INSTALL_FULL_LIBDIR, will have no effect on the build if it is not being used as a variable in cmakelists.txt/in other places. I've seen very few software use the FULL paths in pkgconfig or cmakelists.

Reply:
CMAKE_INSTALL_LIBDIR is more universally used, there is also stuff that uses it wrongly as an absolute path too.

Reply:
Hmm that is unfortunate.

Anyway this PR is fine, that was a pre-existing issue.